esp/mlrs-wifi-bridge/mlrs-wifi-bridge-boards.h

```diff
@@ -215,6 +228,44 @@ GPIO15 = RTC_GPIO13
     #define SERIAL_INVERT true
 
 
+//-- M5Stack M5Stamp C3U Mate
+#elif defined MODULE_M5STAMP_C3U_MATE // ARDUINO_ESP32C3_DEV, ARDUINO_BOARD == ESP32C3_DEV
+    #ifndef ARDUINO_ESP32C3_DEV // ARDUINO_BOARD != ESP32C3_DEV
+	      #error Select board ESP32C3_DEV!
+    #endif
+
+    #undef USE_SERIAL_DBG1
+    #define USE_SERIAL1_DBG
+
+    #undef LED_IO
+    #define USE_LED
+    #include <Adafruit_NeoPixel.h>
+    #define NUMPIXELS  1
+    Adafruit_NeoPixel pixels(NUMPIXELS, 2, NEO_GRB + NEO_KHZ800);
+
+    void led_init(void)
+    {
+        pixels.begin();
+        pixels.setBrightness(20); // not so bright
+    }
+
+    void led_on(void)
+    {
+        pixels.fill(0xFF0000); // red
+        pixels.show();
+    }
+
+    void led_off(void)
+    {
+        pixels.fill(0x000000); // off
+        pixels.show();
+    }
+
+    #define SERIAL_RXD  1 // = RX1
+    #define SERIAL_TXD  0 // = TX1
```

Comment:
interesting
these are the pads labelled porta G1,G0, aren't they?
maybe one could put down a respective comment, I see you referred to IO1/IO0, maybe one could add how they are labelled in M5 dialect